### DISCUSSION TITLE
How to change fonts style  and text color in Blazority

### DISCUSSION THREAD

**Discussion by NattyZelB** [blazority/support#133] [General]
Hi!
 How to change fonts style  and text color in Blazority?, because  i found that my site is not so sharp.Thanks for helping.

**Comment by karan-kang** [comment]
You can modify the font and text-color globally as follows:

```
[cds-theme] {
    --clr-font: cursive;            // this modifies font family
    --cds-global-base: 18;     // this modify font size
}
```

Text color is more complex and controlled by individual controls. All of them still uses one of the following typogrpahy variables.
So, modifying the following set will make changes everywhere.

```
[cds-theme] {
    --cds-alias-typography-color-100: var(--cds-global-color-white);
    --cds-alias-typography-color-200: var(--cds-global-color-construction-600);
    --cds-alias-typography-color-300: var(--cds-global-color-construction-800);
    --cds-alias-typography-color-400: var(--cds-global-color-construction-900);
    --cds-alias-typography-color-450: var(--cds-global-color-construction-1000);
    --cds-alias-typography-color-500: var(--cds-global-color-black);
  }
```
 
 This CSS should be loaded via your index.html or _Host.cshtml after loading the Blazority's base stylesheet
